### PR TITLE
add 2 interop tests

### PR DESCRIPTION
**Pull request by yangzhouhan**

### CODE REVIEW

**Comment by dsymonds** [comment]
Looks like you've picked up a Vim swap file. (I think you'll want to rewrite the commits to get rid of that, not just stack another commit to delete it, since it can hold sensitive information.)

**Comment by yangzhouhan** [comment]
I could find a way to remove a commit history from a PR, would you pls
advise me if you know how to do that? Thanks.

**Comment by yangzhouhan** [comment]
s/could/could not/

**Comment by dsymonds** [comment]
Simplest is to probably add a new commit, then squash the whole series. http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html is a simple approach.

File: interop/client/client.go [hunk 1]
```diff
@@ -245,6 +248,40 @@ func doPingPong(tc testpb.TestServiceClient) {
...
+	pl := newPayload(testpb.PayloadType_COMPRESSABLE, 27182)
+	req := &testpb.StreamingOutputCallRequest{
+		ResponseType: testpb.PayloadType_COMPRESSABLE.Enum(),
+		Payload:      pl,
+	}
+	if err := stream.Send(req); err != nil {
+		grpclog.Fatalf("%v.Send(%v) = %v", stream, req, err)
+	}
+	if _, err := stream.Recv(); grpc.Code(err) != codes.DeadlineExceeded {
+		grpclog.Fatalf("%v got the error %v, want error code: %d", stream, err, codes.DeadlineExceeded)
```
Comment (iamqizhao) [inline interop/client/client.go]: %v.Recv() = _, %v, want error code %d

**Pull request merged**